docs/LangSpec.md

```diff
@@ -138,32 +138,32 @@ The following function declarations are automatically imported into any script:
 
 ```
 /** Returns true if all the conditions are true */
-fun allOf(conditions: Array[Boolean]): Boolean
+def allOf(conditions: Array[Boolean]): Boolean
```

Comment:
@ergomorphic I suggest using `def` since `fun` is removed. Or we can omit it altogether.

Reply:
@greenhat let's use def